document-service/doc-api/src/doc_api/utils/logging.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Centralized setup of logging for the service.
+    Usage:
+        from logging import logger
+
+        logger.info("logging message")
+        # You can pass extra data
+        logger.info(
+            "another logging message",
+            additional={"key1": 1, "key2": {"company": "sample"}, "key3": [1, 2, 3]}
+        )
+        logger.info("another logging message", additional="Yo!!")
+    https://github.com/mfkessai/opentelemetry-python-sample-app/blob/main/flask-on-cloud-functions/custom_loggers.py
+"""
```

Comment:
i see you've pulled in the structured logging, but taken a different one?
struct log is from google cloud run, to do json logging into cloud logging.
do we need to update the package?

Reply:
Copied as is from Patrick's notify-api. Leave for now?